Gifski/Gifski.swift

```diff
@@ -133,7 +135,12 @@ final class Gifski {
 				try g.write(path: conversion.output.path)
 				completionHandler?(nil)
 			} catch {
-				completionHandler?(.writeFailed(error as! GifskiWrapperError))
+				// TODO: Figure out how to not get a write error when the process was simply cancelled
```

Comment:
I'm not seeing any write error.

Reply:
<img width="1841" alt="screenshot 2019-01-18 at 09 29 42" src="https://user-images.githubusercontent.com/225410/51374768-47353800-1b04-11e9-9f71-68e0f8b221e8.png">

Even with the cancellation handler on line 83 of `Gifski.swift` implemented, I still trigger this fatal error without the if statement in place.